module4/lessons/Week4/DependencyInjection.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: page
+title: Dependency Injection
+---
+
+## Introduction
+
+After two days of self-guided learning in pairs, today you will have the opportunity to practice this skill individually. Of course, you can still reach out to teammates to talk through things, but you will be driving the research.
+
+With that in mind, today's lesson on dependency injection is divided into 3 parts.

```

Comment:
with peer feedback, I think this is 4 parts :)

Reply:
Good call! There were originally 3 parts, then I added peer feedback 🙃